seed/challenges/04-data-visualization/json-apis-and-ajax.json

```diff
@@ -30,14 +30,12 @@
     },
     {
       "id": "587d7fad367417b2b2512be1",
-      "title": "Trigger click Events with JavaScript",
+      "title": "Handle Click Events with Javascript using the onclick property",
       "description": [
-        "This section covers how to get data from APIs. APIs - or Application Programming Interfaces - are tools that computers use to communicate with one another.",
-        "You'll learn how to update HTML with the data we get from these APIs using a technology called AJAX.",
         "You want your code to execute only once your page has finished loading. For that purpose, you can attach a JavaScript event to the document called <code>DOMContentLoaded</code>. Here's the code that does this:",
         "<blockquote>document.addEventListener('DOMContentLoaded',function() {<br><br>});</blockquote>",
-        "Next, you can implement a click event handler that goes inside of the <code>DOMContentLoaded</code> function by adding the following code:",
-        "<code>document.getElementById('getMessage').onclick=function(){};</code>",
+        "You can implement event handlers that goes inside of the <code>DOMContentLoaded</code> function. You can implement an <code>onclick</code> event handler which triggers when the user clicks on the element with id <code>getMessage</code>, by adding the following code:",
```

Comment:
Hi @QuincyLarson , thanks for reviewing this PR! Should it be "You can implement event handlers **that** go .." or "You can implement event handlers **which** go .."?